xcat-inventory/xcclient/inventory/backend.py

```diff
@@ -0,0 +1,695 @@
+#!/usr/bin/python
```

Comment:
Please include the head of the copyright

Reply:
fixed